frontend/packages/rhoas-plugin/src/topology/components/TopologyKafkaPanel.tsx

```diff
@@ -49,12 +56,25 @@ export const ConnectedTopologyRhoasPanel: React.FC<TopologyRhoasPanelProps> = ({
     setShowAlert(false);
   };
 
+  const kindRef = referenceFor(akc);
+  const kindObj = modelFor(kindRef);
+
   return (
     <div className="overview__sidebar-pane resource-overview">
       <div className="overview__sidebar-pane-head resource-overview__heading">
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name co-resource-item">
-            <h3>{t('rhoas-plugin~Kafka Connection')}</h3>
+            <ResourceIcon className="co-m-resource-icon--lg" kind={kindRef} />
+            <Link
+              to={resourcePath(
+                kindObj.crd ? kindRef : akc.kind,
```

Comment:
While this works we are triggering this view always for the akc so using akc.kind directly would have been ok as well.